playbooks/templates/cluster_admin_crb.yml.j2

```diff
@@ -0,0 +1,19 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: oidc-cluster-admin2
+  namespace: kube-system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+{% if cluster_admin_users is defined and cluster_admin_users|length > 0 %}
+subjects:
+{% for user in cluster_admin_users %}
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: {{ user }}
+{% endfor %}
```

Comment:
Maybe we should add another `cluster_admin_groups` as well so we have group support from the start

Reply:
Sure, I can add that as well.
Though note that we will not get group support in sc for kubeconfig, because of how googles group support works.

Reply:
Still good to have in wc and maybe for others. So It's good to add regardless.

Reply:
I have now thought about this another round, and it's really bad that we don't get groups with this setup. I would almost then have the process of switching back to dex after the bootstrap just so that we get support for this.

Reply:
After a discussion on slack we decided that it's ok to proceed without group support in sc for now, and discuss this again later.